lib/arrayutils/slice.go

```diff
@@ -0,0 +1,16 @@
+package arrayutils
+
+// This method is used to return uncommon elements between the two slices. It returns (A-B) operation
```

Comment:
// **This method is used to** _find the difference between the two slices and_ **return the uncommon elements in the first set. It returns (A-B) operation**

Reply:
Done